assets/components/odf-lvm/topolvm-controller_deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       - command:
         - /topolvm-controller
         - --cert-dir=/certs
-        image: registry.redhat.io/odf4/odf-topolvm-rhel8@sha256:bd9fb330fc35f88fae65f1598b802923c8a9716eeec8432bdf05d16bd4eced64
+        image: {{ .ReleaseImage.odf_topolvm_rhel8 }}
```

Comment:
Why does the image name hardcode an OS version (`_rhel8`)? Is the intention to have different images for RHEL8 and RHEL9 eventually?

Reply:
I can't speak to why ODF embedded the OS version in the image name, but I'll drop it from field.